db/models.py

```diff
@@ -49,4 +57,52 @@
     )
 
     def __str__(self) -> str:
-        return f"{self.movie.title} {str(self.show_time)}"
+        return f"{self.movie.title} {self.show_time}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(to=get_user_model(),
+                             on_delete=models.CASCADE,
+                             related_name="orders")
+
+    class Meta:
+        ordering = ("-created_at",)
```

Comment:
According to the checklist, 'Implement `class Meta` only after you defined all attributes.' In this case, `class Meta` is correctly placed after all attributes in the `Order` model.